src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -964,12 +951,12 @@ public void VersionHandshake(NetworkPeerRequirement requirements, CancellationTo
                     return;
                 }
 
-                this.SendMessageAsync(new VerAckPayload());
+                this.SendMessageVoidAsync(new VerAckPayload());
                 listener.ReceivePayload<VerAckPayload>(cancellationToken);
                 this.State = NetworkPeerState.HandShaked;
                 if (this.Advertize && this.MyVersion.AddressFrom.Address.IsRoutable(true))
                 {
-                    this.SendMessageAsync(new AddrPayload(new NetworkAddress(this.MyVersion.AddressFrom)
+                    this.SendMessageVoidAsync(new AddrPayload(new NetworkAddress(this.MyVersion.AddressFrom)
```

Comment:
Is this correct?
Sending an empty address list (well just with our nodes address)